website/docs/r/backup_compliance_policy.html.markdown

```diff
@@ -0,0 +1,181 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: backup_compliance_policy"
+sidebar_current: "docs-mongodbatlas-resource-backup-compliance-policy"
+description: |-
+    Provides a Backup Compliance Policy resource.
+---
+# Resource: mongodbatlas_backup_compliance_policy
+
+`mongodbatlas_backup_compliance_policy` provides a resource that enables you to setup a Backup Compliance Policy resource. [Backup Compliance Policy ](https://www.mongodb.com/docs/atlas/backup/cloud-backup/backup-compliance-policy) prevents any user, regardless of role, from modifying or deleting specific cluster settings, backups, and backup configurations. When enabled, the Backup Compliance Policy will be applied as the minimum policy for all clusters and backups in the project. It can only be disabled by contacting MongoDB support. This feature is only supported for cluster tiers M10+.
+When enabled, the Backup Compliance Policy will be applied as the minimum backup policy to all clusters in a Project and will protect all existing snapshots. This will prevent any user, regardless of role, from modifying or deleting existing snapshots prior to expiration. Changes made to existing backup policies will only apply to future snapshots.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+-> **IMPORTANT NOTE:** Once enable a Backup Compliance Policy, no user, regardless of role, can disable the Backup Compliance Policy via Terraform, or any other method, without contacting MongoDB support.   This means that once enabled some resources defined in Terraform will no longer be modifiable.   See the full list of [Backup Compliance Policy Prohibited Actions and Considerations](https://www.mongodb.com/docs/atlas/backup/cloud-backup/backup-compliance-policy/#configure-a-backup-compliance-policy)
+
```

Comment:
```suggestion

-> **NOTE:** With Backup Compliance Policy enabled, cluster backups are retained after a cluster is deleted and backups can be used normally until retention expiration. When Backup Compliance Policy is not enabled, the default, Atlas deletes the cluster's associated backup snapshots when a cluster is terminated. For more details see [Back Up, Restore, and Archive Data](https://www.mongodb.com/docs/atlas/backup-restore-cluster/). 

```

Reply:
Suggest (on top of Zuhair's suggestion) removing "the default" and adding that as its own sentence:

When the Backup Compliance Policy is not enabled, Atlas deletes the cluster's associated backup snapshots when a cluster is terminated. By default, a Backup Compliance Policy is not enabled."